metabase/metabase
# Do not rerun query when navigating from model page to edit model query page

Resolves #24821

### How to Test

1. Visit a model page
2. Inside the `…` menu on the top right, click on "Edit query definition"

Results visualization should be rendered immediately.
No "Doing Science…"

1. Load model query definition page directly on the browser

Query results should load.
